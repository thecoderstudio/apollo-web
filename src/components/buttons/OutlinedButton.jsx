import Button from './Button';
import styled from 'styled-components';

const OutlinedButton = styled(Button)`
  background-color: transparent;
  border: 1px solid ${props => props.theme.accent};
`;

export default OutlinedButton