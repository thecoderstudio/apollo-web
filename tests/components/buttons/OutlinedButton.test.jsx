import React from 'react';
import renderer from 'react-test-renderer';
import OutlinedButton from '../../../src/components/buttons/OutlinedButton'

it('renders correctly', () => {
  const tree = renderer
    .create(<OutlinedButton>Test</OutlinedButton>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
