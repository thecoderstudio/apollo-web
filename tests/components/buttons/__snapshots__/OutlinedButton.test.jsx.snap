// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  max-height: 50px;
  padding: 12px;
  border: none;
  border-radius: 5px;
  color: white;
  font-family: 'B612',sans-serif;
  font-size: 1rem;
  font-weight: 700;
  cursor: pointer;
}

.c0:hover {
  opacity: 0.90;
}

.c0:focus {
  outline: none;
}

.c0:active {
  opacity: 0.80;
}

.c1 {
  background-color: transparent;
  border: 1px solid;
}

<button
  className="c0 c1"
>
  Test
</button>
`;
